composer.json

```diff
@@ -1,22 +1,33 @@
 {
     "name": "youwe/coding-standard-magento2",
     "description": "Youwe PHP Coding Standard for Magento2",
-    "type": "phpcs-sniffs",
-    "homepage": "https://github.com/YouweGit/coding-standard-magento2",
     "license": "proprietary",
```

Comment:
Not blocking, but maybe a good idea to change this to a different license in the future as the code is open source and not proprietary.